### DISCUSSION TITLE
ROS Wrapper for Intel® RealSense™ cameras (Build 4.56.3)

### DISCUSSION THREAD

**Discussion by Nir-Az** [realsenseai/realsense-ros#3269] [General]
![](https://placehold.co/95x20/yellow/black?font=open-sans&text=BETA) ![](https://placehold.co/45x20/green/white?font=open-sans&text=ROLLING) ![](https://placehold.co/45x20/green/white?font=open-sans&text=JAZZY) ![](https://placehold.co/45x20/green/white?font=open-sans&text=IRON) ![](https://placehold.co/45x20/green/white?font=open-sans&text=HUMBLE) ![](https://placehold.co/45x20/green/white?font=open-sans&text=FOXY)

_This is a Beta release_

## What's Changed
* Disabling FPS & TF tests for ROS-CI by @kadiredd in https://github.com/IntelRealSense/realsense-ros/pull/3178
* Update workflows by @SamerKhshiboun in https://github.com/IntelRealSense/realsense-ros/pull/3140
* Retry finding devices with Ykush reset by @kadiredd in https://github.com/IntelRealSense/realsense-ros/pull/3200
* Replace plugins versions withe commit hash by @Noy-Zini in https://github.com/IntelRealSense/realsense-ros/pull/3212
* Hw_reset implementation by @PrasRsRos in https://github.com/IntelRealSense/realsense-ros/pull/3216
* Fix config typo by @patrickwasp in https://github.com/IntelRealSense/realsense-ros/pull/3221
* Suppot a new APIs for motion, accel and gyro streams by @SamerKhshiboun in https://github.com/IntelRealSense/realsense-ros/pull/3225
* Update CMakeLists.txt - remove find_package(fastrtps REQUIRED) by @SamerKhshiboun in https://github.com/IntelRealSense/realsense-ros/pull/3239

## New Contributors
* @Noy-Zini made their first contribution in https://github.com/IntelRealSense/realsense-ros/pull/3212
* @patrickwasp made their first contribution in https://github.com/IntelRealSense/realsense-ros/pull/3221

**Full Changelog**: https://github.com/IntelRealSense/realsense-ros/compare/4.56.1...4.56.3

<hr /><em>This discussion was created from the release <a href='https://github.com/IntelRealSense/realsense-ros/releases/tag/4.56.3'>ROS Wrapper for Intel® RealSense™ cameras (Build 4.56.3)</a>.</em>

**Comment by Shailesh-Pawar-12** [comment]
Im getting build issue in realsense2_camera package 
![Screenshot from 2025-03-21 18-13-35](https://github.com/user-attachments/assets/b6dd37d9-80a5-4191-b234-038670491ba1)
